go.mod

```diff
@@ -3,25 +3,38 @@ go 1.20
 module github.com/cosmos/ibc-go/v7
 
 require (
-	cosmossdk.io/api v0.3.1
-	cosmossdk.io/errors v1.0.0-beta.7
+	cosmossdk.io/api v0.4.2
+	cosmossdk.io/client/v2 v2.0.0-20230615173548-5385116abac4
+	cosmossdk.io/core v0.8.0
+	cosmossdk.io/depinject v1.0.0-alpha.3
+	cosmossdk.io/errors v1.0.0-beta.7.0.20230524212735-6cabb6aa5741
+	cosmossdk.io/log v1.1.0
 	cosmossdk.io/math v1.0.1
+	cosmossdk.io/store v0.1.0-alpha.1.0.20230606190835-3e18f4088b2c
+	cosmossdk.io/tools/confix v0.0.0-20230613133644-0a778132a60f
+	cosmossdk.io/tools/rosetta v0.2.1-0.20230613133644-0a778132a60f
+	cosmossdk.io/x/circuit v0.0.0-20230615173548-5385116abac4
+	cosmossdk.io/x/evidence v0.0.0-20230609160955-4ad71e570d16
+	cosmossdk.io/x/feegrant v0.0.0-20230609160955-4ad71e570d16
+	cosmossdk.io/x/nft v0.0.0-20230616095831-277d5fd37107
+	cosmossdk.io/x/tx v0.8.0
```

Comment:
for reference, I opened https://github.com/cosmos/ibc-go/issues/3968. We don't need to do anything in this pr, we can deal with any agreed upon changes in a followup

Reply:
it's a good idea